README.md

```diff
@@ -106,6 +106,25 @@ To generate a new API token in Crowdin, go to your Account Settings.
 
 ## Setup
 
+### OAuth
+
+1. Install the *Crowdin* plugin using one of the following methods:
```

Comment:
Maybe it would be better to create the `### Installation` section as the first setup step to avoid duplication?